testing/regress/ecl/persist_refresh.ecl

```diff
@@ -0,0 +1,67 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+import Std;
+import Std.File AS FileServices;
+
+//
+//version persistRefresh=true
+//version persistRefresh=false
+
+import ^ as root;
+
+string OriginalTextFilesIp := '.' : STORED('OriginalTextFilesIp');
+string OriginalTextFilesEclPath := '' : STORED('OriginalTextFilesEclPath');
+string OriginalTextFilesOsPath := '' : STORED('OriginalTextFilesOsPath');
+
+persistRefresh := #IFDEFINED(root.persistRefresh, true);
+
+#IF (persistRefresh=true)
+InputFile := OriginalTextFilesOsPath + '/download/europe1.txt';
+#ELSE
+InputFile := OriginalTextFilesOsPath + '/download/europe2.txt';
+#END
```

Comment:
I don't think that is necessary.  We only need tests in one of the two places - and this is testing the general version of the syntax.  What would adding another test cover that this one doesn't?  The only example i can think of is error cases - e.g. a non constant argument.